services/QuillLMS/client/app/bundles/Connect/components/focusPoints/focusPointsContainer.jsx

```diff
@@ -133,6 +133,12 @@ export class FocusPointsContainer extends Component {
     this.setState({focusPoints: focusPoints})
   }
 
+  handleNameChange = (e, key) => {
+    const { focusPoints } = this.state
+    focusPoints[key].name = e.target.value
+    this.setState({focusPoints: focusPoints})
```

Comment:
No need to change it here but for future reference, you can just do `this.setState({ focusPoints, })`